README.md

```diff
@@ -52,6 +52,31 @@ If you don't want to enter `dev-env` manually each time using `eval "$(dev-env/b
 you can also install [direnv](https://direnv.net). This repo already provides a `.envrc`
 file, with an option to add more in a `.envrc.private` file.
 
+#### Mac
+
+On Mac `dev-env` can be installed with:
+
+1. Install Nix by running: `bash <(curl -sSfL https://nixos.org/nix/install)`
+   Note that Nix recommends a *multi-user installation* on macOS. You need to configure `/etc/nix/nix.conf` To use Nix caches.
```

Comment:
It's no longer a recommendation (it's been "recommended" for year and is recommended on Linux too), it's the only option with recent versions of nix.

Reply:
Fixed.